README.md

```diff
@@ -58,3 +58,15 @@ const WrappedComponent = withPromises({
   anotherPromiseData: Promise.resolve('random data'),
 })(MyComponent);
 ```
+
+You can also pass a function to withPromise, so that you are able to fetch multiple times as the component is updated. The above example would be modified to look like this:
+
+```js
+const WrappedComponent = withPromises((props, oldProps, oldPromises){
```

Comment:
I think you missed the `function` keyword or to use the fat-arrow like
```js
withPromises((props, oldProps, oldPromises) => ({ /* the promises object */ })(MyComponent)
```

Reply:
oops...

Reply:
Do you prefer to do the modification or that I do it after merging your PR?